sphinx/about/whats-new.rst

```diff
@@ -1,6 +1,58 @@
 Version history
 ===============
 
+6.9.0 (2022 February)
+---------------------
+
+New file formats:
+
+* Leica LOF
+   - added support for reading Leica LOF files.
+     This functionality was implemented with thanks to Leica Microsystems
+
+* Leica XLEF
+   - added support for reading Leica XLEF files.
+     This functionality was implemented with thanks to Leica Microsystems
+
+File format fixes and improvements:
+
+* Aperio SVS TIFF
+   - fixed handling of macro and label images
```

Comment:
I thought this was https://github.com/ome/bioformats/pull/3764 but it looks like this got excluded and pushed away?

Reply:
Yeah, apparently I only removed it from the website PR for some reason. Corrected now